user/languages/erlang.md

```diff
@@ -24,31 +24,37 @@ otp_release:
   - R16B03
 ```
 
-Get a complete list of the pre-compiled versions available on the VM by adding `kerl list installations` to the `before_script:` section of your `.travis.yml`. Note that this list does *not* include releases which are downloaded on demand, such as 18.1 .  
+Get a complete list of the pre-compiled versions available on the VM by adding `kerl list installations` to the `before_script:` section of your `.travis.yml`. Note that this list does *not* include releases which are downloaded on demand, such as 18.1.
 
 ## Default Test Script
 
-Travis CI by default assumes your project is built using [Rebar](https://github.com/rebar/rebar) and uses EUnit. The exact command Erlang builder will use by default is
+Travis CI by default assumes your project is built using [Rebar3](https://github.com/erlang/rebar3) and uses EUnit. The exact command Erlang builder will use by default is
 
 ```bash
-rebar compile && rebar skip_deps=true eunit
+rebar3 compile && rebar3 skip_deps=true eunit
 ```
 
-if your project has `rebar.config` or `Rebar.config` files in the repository root. If this is not the case, Erlang builder will fall back to
+if your project has `rebar.config` or `Rebar.config` files in the repository root.
+
+On older images where `rebar3` is not available, we fall back to [`rebar`](https://github.com/rebar/rebar).
+
+If this is not the case, Erlang builder will fall back to
 
 ```bash
 make test
 ```
 
 ## Dependency Management
 
-The Erlang builder on travis-ci.org assumes [Rebar](https://github.com/basho/rebar) is used for dependency management, and runs
+The Erlang builder on travis-ci.org assumes Rebar3 is used for dependency management, and runs
 
 ```bash
-rebar get-deps
+rebar3 get-deps
```

Comment:
This is also no longer needed, it is done automatically when `rebar3 compile` runs (or any other command, such as  `eunit`)